package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "webtorrent",
   "description": "Streaming torrent client",
-  "version": "0.96.5",
+  "version": "0.97.0",
```

Comment:
I did use `npm version minor`

I bumped the version because I need it for the desktop app: https://github.com/feross/webtorrent-desktop/pull/913